tests/test_helpers/pivot_helpers.ts

```diff
@@ -2,11 +2,75 @@ import { DispatchResult, Model, UID } from "../../src";
 import { toZone } from "../../src/helpers";
 import { SpreadsheetPivotCoreDefinition } from "../../src/types/pivot";
 import { pivotModelData } from "../pivots/pivot_data";
+import { setCellContent } from "./commands_helpers";
 
 export function createModelWithPivot(range: string): Model {
   return new Model(pivotModelData(range));
 }
 
+/**
+ * Creates a Model instance with a predefined pivot table setup.
+ * It contains a "PIVOT" formula in the cell A8
+ *
+ */
+export function modelWithPivot() {
```

Comment:
I'm half-half excited about this. It usually leads to tests that are hard to read and understand a you don't know what the pivot looks like when reading the test.
I personally prefer to manually setup the smallest pivot possible in each test, even if it takes more lines.
What's your opinion ?

Reply:
> you don't know what the pivot looks like when reading the test.
Without this kind of helper, it's a mess to create every single pivot test :/
I'm not 100% for, but I think it's necessary.
The setup a pivot is not straightforward. It requires at least 15-20 lines of codes, that reduces the readability of the code IMO.
I want to add an helper to print the pivot, something similar to `printDebugModel` to help us.

Reply:
11 lines :stuck_out_tongue: 
```ts
    const grid = {
      A1: "Date",       B1: "Price",
      A2: "2024-12-31", B2: "10",
      A3: "1995-04-14", B3: "30",
    };
    const model = createModelFromGrid(grid);
    addPivot(model, "A1:B3", {
      columns: [],
      rows: [{ name: "Date", granularity: "year_number" }],
      measures: [{ name: "Price", aggregator: "sum" }],
    });
```

> I want to add an helper to print the pivot, something similar to printDebugModel to help us.

could be helpful :)

Reply:
Mmm I didn't know `createModelFromGrid` :o 
Ok then I will use that, it's not painful now. I'll see in my task to move the tests from odoo if we need it

Reply:
Updated, and I created `printPivot` helper ;-)